lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -2766,10 +2766,6 @@ SPIRVValue *LLVMToSPIRVBase::oclTransSpvcCastSampler(CallInst *CI,
                                                      SPIRVBasicBlock *BB) {
   assert(CI->getCalledFunction() && "Unexpected indirect call");
   llvm::Function *F = CI->getCalledFunction();
-  auto FT = F->getFunctionType();
```

Comment:
Same concern as above. Why is this change par of this PR? Is this PR accomplishing more that what the description says?

Thanks